onnx/defs/nn/defs.cc

```diff
@@ -1542,4 +1534,73 @@ ONNX_OPERATOR_SET_SCHEMA(
         })
         .SetDoc(TfIdfVectorizer_ver9_doc));
 
+static const char* StringNormalizer_ver10_doc = R"DOC(
+StringNormalization performs string operations for basic cleaning. 
+This operator has only one input (denoted by X) and only one output 
+(denoted by Y). This operator first examines the elements in the X, 
+and remove elements specified in "stopwords" attribute. 
+Note that an implementation should sequentially remove "stopwords[0]," then "stopwords[1]," 
```

Comment:
"Note that an implementation should sequentially remove "stopwords[0]," then "stopwords[1]," "
why? is this because that the stop word may not be unigram?

Reply:
Right. It's for avoiding undefined behavior in that case.

Reply:
I discussed with Dmitri offline. Looks to us the order/sequantial does not matter here. Say, input is "x", "y", "z", and stop words are "xy", "yz", then nothing will be removed from input.

Reply:
Ah, yeah. Makes sense. Thank you.